### ISSUE TITLE
Improper "VA" kerning (small caps)

### ISSUE THREAD

**Issue by gdonval** [silnrsi/font-gentium#32]
Example (typeset with Lualatex), smallcaps on the left, caps on the right:

![image](https://github.com/silnrsi/font-gentium/assets/2644113/90684284-853a-4a14-9da3-2619e4cb1469)

I'd argue that in both forms ON could be a bit closer and even in all caps, AV is still a bit far but in small caps AV kerning looks plain non-existent.

**Comment by jvgaultney** [comment]
We won't be changing the ON spacing, as that's rather fundamental to the overall spacing. Here's a preview of v7, with slightly tightened VA: 

![image](https://github.com/silnrsi/font-gentium/assets/4226750/60586335-a4fb-4c14-874d-9f9484360fe6)

That may not look like enough at large sizes (it's not), but at text sizes it seems to be enough.

**Comment by gdonval** [comment]
No worries for ON, it's nowhere near as distracting than the missing kerning in-between VA (which is why the title only mentions VA). Are small-caps also covered by those changes?

**Comment by jvgaultney** [comment]
Yes - the small caps will inherit the same relative kerning as the caps.

**Comment by jvgaultney** [comment]
Improved for the next version
